### ISSUE TITLE
Module Not Found Error 

### ISSUE THREAD

**Issue by filiatalya7n8vx** [Z3zrnSOLANA/SOLANA_SNIPER_BOT#1]
Howdy,

I am receiving this error when running my code:

Traceback (most recent call last):
File "C:\Users\bean\OneDrive\Desktop\python.py", line 75, in
from solana.account import Account
ModuleNotFoundError: No module named 'solana.account'

Any suggestions on how to get past this issue? I've tried updating pip and installing solana-py from the source code. I've also tried searching for the sub-module in the library, has the structure changed?

cheers!

**Comment by Z3zrnSOLANA** [comment]
Yes. you can try `pip install -r requirements.txt`  before run main.py

 Please use https://kevinheavey.github.io/solders/, its new version solana libs
